src/middlewared/middlewared/plugins/system_/dmi.py

```diff
@@ -7,6 +8,8 @@ class SystemService(Service):
     # DMI information is mostly static so cache it
     HAS_CACHE = False
     CACHE = {
+        'bios-release-date-raw': '',
+        'bios-release-date-formatted': '',
```

Comment:
It was `formatted` (in an unknown format) before you parsed it. I would say now it is `parsed` (as ZFS properties), isn't it?
And will we ever need this date as raw?